subprojects/core/src/test/groovy/org/gradle/api/internal/project/taskfactory/AnnotationProcessingTasks.java

```diff
@@ -130,6 +132,13 @@ public TaskWithIncrementalAction(Action<IncrementalTaskInputs> action) {
         public void doStuff(IncrementalTaskInputs changes) {
             action.execute(changes);
         }
+
+        @Optional
+        @OutputFiles
```

Comment:
Why not use `@OutputFile` here?

Reply:
No good reason. Changed to `@OutputFile` so it is in line with the name.